organisaatio-ui/src/tools/mappers.ts

```diff
@@ -37,3 +42,33 @@ export const checkHasSomeValueByKieli = (KielisetYhteystiedot: Yhteystiedot[Supp
             .length > 0
     );
 };
+
+export const findCurrentNimi = (nimet: UiOrganisaationNimetNimi[], nimi: Nimi) => {
+    const currentNimi = nimet.find((nimetNimi) => JSON.stringify(nimetNimi.nimi) === JSON.stringify(nimi));
+    const editModeDisabled =
+        currentNimi && !!nimet.find((nimetNimi) => new Date(nimetNimi.alkuPvm) > new Date(currentNimi.alkuPvm));
```

Comment:
`Date.after`?
I think it would be more consistent to use one approach for instantiating dates, either vanilla or moment. maybe not both? (could be that I have missed something here)